theme/src/templates/page.js

```diff
@@ -2,13 +2,15 @@ import React, { Component } from "react"
 import { graphql } from "gatsby"
 import Img from "gatsby-image"
 import PageFlipper from "../components/PageFlipper"
+import SEO from "../components/seo"
 
 export default class postLayout extends Component {
   render() {
     const { file } = this.props.data
 
     return (
       <>
+        <SEO />
```

Comment:
There should be a title prop added here set to equal `file.name`.

Reply:
page.js
`<title titleProp={file.name} />`